.github/workflows/test.yml

```diff
@@ -1,38 +1,49 @@
-on: push
+on:
+  push:
+  pull_request:
+    types: [synchronize]
 name: Test
 jobs:
   test:
     strategy:
       fail-fast: false
       matrix:
-        go-version: [1.16.x, 1.17.x]
+        go-version: [1.17.x, 1.18.x]
```

Comment:
What do you think about `1.x`?

Reply:
Does that work? What versions would it map to?

Reply:
It works, and maps to the most-recent version, modulo some lag in updating the action. Although, upon reflection, if we commit to supporting the most-recent and next-most-recent version, it doesn't help much there. Maybe not a good idea.

Reply:
OK. I vote we keep an explicit version list here then.